pennylane/devices/default_qubit_jax.py

```diff
@@ -41,9 +41,9 @@ def _validate_jax_version():
             "You can either downgrade JAX to version 0.4.3 or update to a more recent version if available.\n"
             "If you downgrade, you will also need to downgrade JAXLIB to version 0.4.3 or earlier.\n"
             "If you are using pip to manage your packages, you can run the following command:\n\n"
-            "\tpip install 'jax==0.4.3' 'jaxlib==0.4.3'\n\n"
+            "\tpip install 'jax' 'jaxlib' -U\n\n"
```

Comment:
Do we need the quotes around `'jax'` and `'jaxlib'`?

Reply:
I think you are right, I removed it!